platform-tools-sdk/cargo-build-sbf/src/main.rs

```diff
@@ -395,8 +392,7 @@ fn main() {
                 .long("sbf-sdk")
                 .value_name("PATH")
                 .takes_value(true)
-                .default_value(&default_sbf_sdk)
-                .help("Path to the Solana SBF SDK"),
+                .help("UNUSED: Path to the Solana SBF SDK."),
```

Comment:
I vaguely remember you mentioning that Nix environments need to manually set the path to the SDK. My memory may be failing though. Isn't that the case anymore?